src/components/task/breadcrumbs.cjsx

```diff
@@ -39,11 +42,46 @@ module.exports = React.createClass
     # until the recovery step has been loaded
     TaskStore.on('task.afterRecovery', @update)
 
+    crumbs = @getCrumableCrumbs()
+
+  componentDidMount: ->
+    @calculateCrumbsWidth()
+
+  calculateCrumbsWidth: (crumbDOM) ->
+    if @isMounted()
+      currentCrumbWidth = 0
+      crumbsWidth = _.reduce(@refs, (memo, ref) ->
+        refDOM = ref.getDOMNode()
+        computedStyle = window.getComputedStyle(refDOM)
```

Comment:
No biggy since we're using this all over the place, but `getDOMNode` is being deprecated in favor of `React.findDOMNode`  At some point we'll have to go through and replace all instances of it.